releasenotes/notes/add-datadog-lambda-integration-520a41fe618280ac.yaml

```diff
@@ -0,0 +1,6 @@
+---
+features:
+  - |
+    Add timeout spans support for ``datadog_lambda>=4.66.0``.
```

Comment:
again, I don't think users will know what "timeout spans" means here. Can we reword (maybe something along the lines of my suggestion above)?

Reply:
Makes sense, updating it.